src/tests/test_addressbook.py

```diff
@@ -0,0 +1,31 @@
+"""
+Tests for addresbook validation
+"""
+# import pdb;pdb.set_trace()
+import unittest
+
+from pybitmessage import addresses, helper_addressbook
+from pybitmessage.bmconfigparser import BMConfigParser
+
+
+class TestAddAddressBook(unittest.TestCase):
+    """this class is used for testting add address feature"""
+
+    addr = 'BM-2cVvkzJuQDsQHLqxRXc6HZGPLZnkBLzEZY'
+
+    def test_add_same_address_twich_in_addressbook(self):
+        """checking same address is added twice in addressbook"""
+        if helper_addressbook.insert(label='test1', address=self.addr):
```

Comment:
this isn't a correct test. It should always add it twice, like I said. No `if`s.
```
assertTrue(insert(label='test1', address=self.addr))`
assertFalse(insert(label='test1', address=self.addr)) # will auto-detect duplicate and fail